.storybook/config.js

```diff
@@ -14,7 +15,7 @@ addParameters({
   }
 })
 
-// a11y
+addDecorator(withInfo)
```

Comment:
Suggestion: add comment to remind team that the info decorator needs to be the first decorator added under config.js